src/main/java/net/atos/zac/policy/output/ZaakRechten.java

```diff
@@ -30,7 +49,26 @@ public ZaakRechten(
             @JsonbProperty("afbreken") final boolean afbreken,
             @JsonbProperty("heropenen") final boolean heropenen,
             @JsonbProperty("wijzigenZaakdata") final boolean wijzigenZaakdata,
-            @JsonbProperty("wijzigenDoorlooptijd") final boolean wijzigenDoorlooptijd
+            @JsonbProperty("wijzigenDoorlooptijd") final boolean wijzigenDoorlooptijd,
+            @JsonbProperty("verlengen") final boolean verlengen,
+            @JsonbProperty("opschorten") final boolean opschorten,
+            @JsonbProperty("hervatten") final boolean hervatten,
+            @JsonbProperty("creeeren_document") final boolean creeerenDocument,
```

Comment:
just thinking that Json properties are not underscore separated, see also `wijzigenDoorlooptijd`, we have to ensure we are consistent, please check !

Reply:
OPA policies have snake-case names, and we previously have done the mapping to camel-case here via the properties. I was following the same concept. 

You are right though that `wijzigenDoorlooptijd` does not follow this. I can amend it to be in snake case if we agree this should be the convention we follow.

@edgarvonk @bas-info-nl WDYT?

Reply:
Moved to snake_case for policy names